language/gpt-j/README.md

```diff
@@ -75,8 +79,7 @@ Please download the fine-tuned GPT-J checkpoint using the instructions below. Th
 The following MLCommons CM commands can be used to programmatically download the model checkpoint. 
 
 ```
-pip install cm4mlops
-cm run script --tags=get,ml-model,gptj,_pytorch,_rclone -j
+cm run script --tags=get,ml-model,gptj,_pytorch,_rclone --outdirname=<path/to/desired/output-directory>
```

Comment:
`--outdirname=./model ` is more appropriate as that's the path expected in the benchmark run command later.

Reply:
@arjunsuresh thanks for the feedback. I will change it.